### PR TITLE
Dh 1929 dag sync composite

### PR DESCRIPTION
**Pull request by aabouelleil-seniorlink**
composite action for syncing airflow dags, to be used by multiple datahub repositories, initially for this PR:
https://github.com/seniorlink-vela/dh-dags/pull/20

### CODE REVIEW

File: composite/datahub/airflow-sync/action.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        AIRFLOW_URL: 'https://dh-airflow.vela.care/home/api/v1/dags/dh_sync_dags_in_s3_to_local_airflow_dags_folder/dagRuns'
+    - name: Get Airflow URL
+      if: inputs.airflow_env != 'prod'
+      env:
+        AIRFLOW_URL: 'https://dh-airflow.${{inputs.airflow_env}}.alwaysreach.net/api/v1/dags/dh_sync_dags_in_s3_to_local_airflow_dags_folder/dagRuns'
+    - name: Sync Airflow Dags
+      id: syncAirflow
+      uses: fjogeleit/http-request-action@v1
+      with:
+        url: $AIRFLOW_URL
```
Comment (aabouelleil-seniorlink) [inline composite/datahub/airflow-sync/action.yml:41]: I'm not sure if this is the correct syntax for accessing an environment variable specified in another step

Comment (reyesjx7) [reply composite/datahub/airflow-sync/action.yml:41]: I haven't done this either but i found this in the github docs: [https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-an-environment-variable](https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-an-environment-variable)

Seems like you need to assign it to the github env to use it in subsequent steps.

Comment (reyesjx7) [reply composite/datahub/airflow-sync/action.yml:41]: Yeah I noticed that too, but I couldn't find another way to do it in the docs. Approving so we can test this.

**Review by reyesjx7 — APPROVED** [review]

File: composite/datahub/airflow-sync/action.yml (hunk 1 above)
Comment (aabouelleil-seniorlink) [reply composite/datahub/airflow-sync/action.yml:41]: I created a sandbox repo for us to test out actions here: 
https://github.com/seniorlink-vela/dh-github-actions-development

I tested the method you linked and it does NOT echo the actual variable to logs, so I will update to try that.

**Pull request merged**